spec/requests/configuration_script_sources_spec.rb

```diff
@@ -32,23 +32,36 @@
   end
 
   describe 'GET /api/configuration_script_sources/:id' do
+    let(:repository) { FactoryBot.create(:configuration_script_source) }
+
     it 'will show a configuration script source with an appropriate role' do
-      repository = FactoryBot.create(:configuration_script_source)
       api_basic_authorize collection_action_identifier(:configuration_script_sources, :read, :get)
-
       get(api_configuration_script_source_url(nil, repository))
 
+      expected = {'href' => api_configuration_script_source_url(nil, repository)}
+
+      expect(response.parsed_body).to include(expected)
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'allows accessing verify_ssl for EmbeddedAnsible::AutomationManager::ConfigurationScriptSource' do
+      embedded_ansible_repository = FactoryBot.create(:embedded_ansible_configuration_script_source)
```

Comment:
Is this specific to EmbeddedAnsible or does it also work on the base class?

Reply:
No, it does not, since nothing else has `:git_repository` as a relation like the specific version does:

https://github.com/ManageIQ/manageiq/blob/5aa0822a056a28e1cd7790768a05343cda835684/app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source.rb#L13

Which was added here:

https://github.com/ManageIQ/manageiq/pull/21298